### ISSUE TITLE
W1 Ch2

### ISSUE THREAD

**Issue by JSchuurmans** [JannesKlaas/MLiFC#4]
Measuring losses:
m=1: y_i -> y_1, yhat_i -> yhat_1, sum_{i=1}^1 -> (delete summation or sum_{i=1})

cross-entropy loss and logistic loss are different loss functions https://en.wikipedia.org/wiki/Loss_functions_for_classification
Don't call logistic loss, log loss. This is can be confused with the way you use log in log likelihood.

define yhat as the predicted value of y

Mathematical representation:
The learning rate alpha looks too much like a, use gamma.

inverse of the gradient -> subtract the gradient (also key takeaways)

**Comment by JCKrick** [comment]
I recommend defining all variables again in the backpropagation section as many variables are defined way earlier.

**Comment by JannesKlaas** [comment]
Updated the sum expression.
